src/logger.sh

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+#
+
+[[ "${_YB_LOGGER_INCLUDED:=""}" == "yes" ]] && return 0
```

Comment:
Again, do not use `yes` / `no` for boolean variables. I think we've established `true`/`false` as the convention.